src/vmshepherd/presets/preset.py

```diff
@@ -83,36 +83,41 @@ def _reset_counters(self):
         """
         if if_older_than is None or time.time() - self._vms_refresh_time > if_older_than:
             self._vms_refresh_time = time.time()
-            self._vms = await self.iaas.list_vms(self.name)
+            self._runtime_vms = await self.iaas.list_vms(self.name)
+        else:
```

Comment:
I couldn't find any tests regarding this change

Reply:
Tests was changed for new logic.